packages/api/src/api/stats/stats.controller.ts

```diff
@@ -18,7 +18,11 @@ export class StatsController {
 
   @Get("/ethprice")
   public async ethPrice(): Promise<EthPriceResponseDto> {
-    const token = await this.tokenService.findOne(ETH_TOKEN.l2Address, { usdPrice: true, offChainDataUpdatedAt: true });
+    const nativeToken = await chainNativeToken();
```

Comment:
It's too expensive to call blockchain each time we need to get the token info. We could cache the response, but I'd recommend to do it via configuration. Let's have additional env vars for BASE TOKEN attributes or a config file.

Reply:
Sure, I'll implement it that way.

Reply:
Still working on it but, I've moved this request to a separate module [here](https://github.com/matter-labs/block-explorer/pull/172/files#diff-f0dabe8e027b83905fee2eff55a4a9794721d2c64b0a7c6cfeb7abd036cf4b06).
The request should be done once when bootstrapping and subsequently return the cached value.
What do you think of this approach @Romsters ?